### ISSUE TITLE
error in nodejs context

### ISSUE THREAD

**Issue by gre** [wbuss/JSONBigNumber#2]
```
json-bignumber/dist/JSONBigNumber.min.js:1
!function(t,e){"object"==typeof exports&&"object"==typeof module?module.exports=e(require("bignumber.js")):"function"==typeof define&&define.amd?define(["bignumber.js"],e):"object"==typeof exports?exports.JSONBigNumber=e(require("bignumber.js")):t.JSONBigNumber=e(t.BigNumber)}(window,function(t){return function(t){var e={};function r(n){if(e[n])return e[n].exports;var o=e[n]={i:n,l:!1,exports:{}};return t[n].call(o.exports,o,o.exports,r),o.l=!0,o.exports}return r.m=t,r.c=e,r.d=function(t,e,n){r.o(t,e)||Object.defineProperty(t,e,{enumerable:!0,get:n})},r.r=function(t){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(t,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(t,"__esModule",{value:!0})},r.t=function(t,e){if(1&e&&(t=r(t)),8&e)return t;if(4&e&&"object"==typeof t&&t&&t.__esModule)return t;var n=Object.create(null);if(r.r(n),Object.defineProperty(n,"default",{enumerable:!0,value:

ReferenceError: window is not defined
    at Object.<anonymous> (json-bignumber/dist/JSONBigNumber.min.js:1:279)
```

**Comment by wbuss** [comment]
I have not been keeping this up to date since the project i created it for was decommissioned.
I think that JSONBigNumber has had some updates that contained some breaking changes so i beleive that is probably the cause of the issues you are seeing. I may get a chance to look at this at some point this week

**Comment by DeployIsComing** [comment]
So if the lib is not updated please remove it from NPM

**Comment by wbuss** [comment]
This issue has been fixed with v1.1.1
[Fix Uncaught ReferenceError](https://github.com/wbuss/JSONBigNumber/commit/7742e4d6ee1cba9a8da3e76f85e16c7178585595)
